doc/classes/SceneTree.xml

```diff
@@ -82,26 +83,25 @@
 				}
 				[/csharp]
 				[/codeblocks]
-				The timer will be automatically freed after its time elapses.
 			</description>
 		</method>
 		<method name="create_tween">
 			<return type="Tween" />
 			<description>
-				Creates and returns a new [Tween].
+				Returns a newly created [Tween].
 			</description>
 		</method>
 		<method name="get_first_node_in_group">
 			<return type="Node" />
 			<param index="0" name="group" type="StringName" />
 			<description>
-				Returns the first node in the specified group, or [code]null[/code] if the group is empty or does not exist.
+				Returns the first [Node] in the specified [param group], or [code]null[/code] if the group does not exist. See also [method get_nodes_in_group].
```

Comment:
Clarify what "first" means (first one added to the group? Or first in the hierarchy?).

From a quick look at the code, these are sorted by `Node.is_greater_than`, so it seems like it's based on position in the hierarchy.

Reply:
It is indeed first in the hierarchy. Although this description should be rewritten more like this PR's `get_nodes_in_group`.